README.md

```diff
@@ -72,24 +72,26 @@ Required options:
 * `hokusai images` - List image metadata in the AWS ECR project repo.
 
 ### Working with Kubernetes
-Hokusai uses `kubectl` to connect to Kubernetes. You first need to make sure `kubectl` is installed and you have proper config setup for connecting to your Kubernetes. Hokusai `deps` commands provide basic setup for this:
+Hokusai uses `kubectl` to connect to Kubernetes. Hokusai `configure` provides basic setup for installing and configuring kubectl:
 ```bash
-hokusai deps --help
+hokusai configure --help
 ```
 Recommended approach is to upload your `kubectl` config to S3 and use following command to install it:
 ```bash
-hokusai deps --s3-bucket <bucket name> --s3-key <file key>
+hokusai configure --kubectl-version --s3-bucket <bucket name> --s3-key <file key>
 ```
 
-When running `hokusai setup` `staging.yml` and `production.yml` are created in the hokusai project directory. These files define what Hokusai calls "stacks", and Hokusai is opinionated about a workflow between a staging and production Kubernetes context.  Hokusai commands such as `stack`, `secrets`, `deploy` and `logs` require invocation with either the `--staging` or `--production` flag, which references the respective stack YAML file and interacts with the respective Kubernetes context.
+When running `hokusai setup` `staging.yml` and `production.yml` are created in the hokusai project directory. These files define what Hokusai calls "stacks", and Hokusai is opinionated about a workflow between a staging and production Kubernetes context.  Hokusai commands such as `stack`, `env`, `deploy` and `logs` require invocation with either the `--staging` or `--production` flag, which references the respective stack YAML file and interacts with the respective Kubernetes context.
 
-### Working with Secrets
+### Working with Environment Variables
 
-* `hokusai secrets get` - Prints secrets stored on the Kubernetes server
-* `hokusai secrets set` - Sets secrets on the Kubernetes server. Secrets are stored for the project as key-value pairs in the Kubernetes Secret object `{project_name}-secrets`
-* `hokusai secrets unset` - Removes secrets stored on the Kubernetes server
+* `hokusai env create` - Create the Kubernetes configmap object `{project_name}-environment`
+* `hokusai env get` - Print envrionment variables stored on the Kubernetes server
+* `hokusai env set` - Set envrionment variables on the Kubernetes server. Environment variables are stored for the project as key-value pairs in the Kubernetes configmap object `{project_name}-environment`
+* `hokusai env unset` - Remove envrionment variables stored on the Kubernetes server
+* `hokusai env delete` - Delete the Kubernetes configmap object `{project_name}-environment`
```

Comment:
the semantics of this are pretty simple and clearer than `secrets` - if i understand correctly, we no longer have to care whether the env var is sensitive or not, so this is nicely unified now.